### PR TITLE
Updates `moped_tags` for Mobility Annual Plan (MAP)

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/10362

## Testing
Local or[ test deployment](https://10362-jc-update-moped-tags--atd-moped-main.netlify.app/moped/session/signin)

**Steps to test:**
1. Navigate to `moped/dev/lookups#moped-tags`
2. Verify this is the complete list of tags:

Type | Name | Slug
-- | -- | --
Work Plan | MAP Signals 2023 | map_signals2023
Work Plan | MAP TIA Fiscal | map_tiafiscal
Work Plan | MAP Transit Enhancement 2023 | map_transitenhancement2023
Work Plan | MAP Transit Speed and Reliability (CapMetro ILA) 2023 | map-transit-speed-and-reliability-capmetro-ila-2023
Work Plan | MAP Vision Zero - Major Safety 2023 | map-vision-zero-major-safety-2023
Work Plan | MAP Vision Zero - Ped Safety 2023 | map-vision-zero-ped-safety-2023
Work Plan | MAP Vision Zero - Rapid Response/Small Scale 2023 | map-vision-zero-rapid-response-small-scale-2023
Work Plan | MAP Vision Zero - Speed Management 2023 | map_vzero_speedmanagement2023
Work Plan | MAP Vision Zero - Street Lighting 2023 | map-vision-zero-street-lighting-2023
Work Plan | MAP Vision Zero - Systemic Safety 2023 | map-vision-zero-systemic-safety-2023
Work Plan | MAP Vision Zero - Systemic Safety Curves 2023 | map_vzero_systemicsafety_curves2023

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-editor/src/queries/tableLookups.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const TABLE_LOOKUPS_QUERY = gql`
         subcomponent_id
       }
     }
-    moped_tags(where: { is_deleted: { _eq: false } }) {
+    moped_tags(where: { is_deleted: { _eq: false } }, order_by: { name: asc }) {
```
Comment (johnclary) [inline moped-editor/src/queries/tableLookups.js:36]: just updating the sort order here to match the Tags component on the project summary page

File: moped-editor/package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "atd-moped-editor",
   "author": "ATD Data & Technology Services",
   "license": "CC0-1.0",
-  "version": "1.10.0",
+  "version": "1.9.1",
```
Comment (johnclary) [inline moped-editor/package.json:5]: we're going to ship this as a patch 🪡

File: moped-database/migrations/1664560448539_update_moped_tags_for_map/down.sql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+DELETE FROM moped_tags WHERE slug = 'map-transit-speed-and-reliability-capmetro-ila-2023';
+DELETE FROM moped_tags WHERE slug = 'map-vision-zero-ped-safety-2023';
+DELETE FROM moped_tags WHERE slug = 'map-vision-zero-rapid-response-small-scale-2023';
+DELETE FROM moped_tags WHERE slug = 'map-vision-zero-street-lighting-2023';
+DELETE FROM moped_tags WHERE slug = 'map-vision-zero-systemic-safety-2023';
+UPDATE moped_tags SET is_deleted = FALSE WHERE slug = 'map_vzero_intersectionspotential2023';
+UPDATE moped_tags SET name = 'MAP Vision Zero Intersections 2023', slug = 'map_vzero_intersections2023' WHERE slug = 'map-vision-zero-major-safety-2023';
+UPDATE moped_tags set name = 'MAP Vision Zero - Systemic Safety Curves 2023' WHERE slug = 'map_vzero_systemicsafety_curves2023';
+UPDATE moped_tags set name = 'MAP Vision Zero - Speed Management 2023' WHERE slug = 'map_vzero_speedmanagement2023';
```
Comment (mddilley) [inline moped-database/migrations/1664560448539_update_moped_tags_for_map/down.sql:9]: We probably won't ever run this down migration, but I think these last two updates are the same as the up migration.

**Review by mddilley — APPROVED** [review]
Looks great to me - all the new and updated tags are present in the DB and show just as described in the test steps. 🙏

**Review by chiaberry — APPROVED** [review]

File: moped-database/migrations/1664560448539_update_moped_tags_for_map/down.sql (hunk 3 above)
Comment (johnclary) [reply moped-database/migrations/1664560448539_update_moped_tags_for_map/down.sql]: dang it! thanks @mddilley for catching that.

**Comment by johnclary** [comment]
[here's a project](https://moped.austinmobility.io/moped/projects/1878?tab=summary) in staging with all the tags. let's see watch what happens when we merge.

Screen Shot 2022-10-05 at 2 07 50 PM

**Pull request merged**